test/fixtures/people_roles.yml

```diff
@@ -0,0 +1,6 @@
+one:
+  person_id: 1
+  role_id: 1
+two:
+  person_id: 2
+  role_id: 1
```

Comment:
I _think_ you might be able change these to 
```
person: one
role: foster
```
etc.

Reply:
That makes a lot of sense, but wasn't working. Since I typically work with Rspec, I didn't want to dig too deep in case there's a setting we need to load fixtures in specific order, or I'm otherwise just missing something. I tried setting `self.pre_loaded_fixtures = true` but that didn't change anything.

For roles specifically, it's possibly because of how Rolify sets up the association. But even with `person_homes.yml`, I was getting `Foreign key violations found in your fixture data` errors.